main.nf

```diff
@@ -83,6 +83,11 @@ if (params.help){
     exit 0
 }
 
+// Check if genome exists in the config file
+if (!params.genomes.containsKey(params.genome)) {
```

Comment:
Needs to also check if `params.genome` is set, as this is only used with iGenomes - people can also specify their own reference paths manually.

Reply:
Right! Thanks, I'll work on that!

Reply:
Already in here https://github.com/nf-core/rnaseq/pull/51

Reply:
OK great! So nothing further needs to be done?